pkg/services/rpcsrv/server_test.go

```diff
@@ -73,30 +73,70 @@ type rpcTestCase struct {
+	// genesisBlockHash is an LE hash of genesis block in basic testing chain.
+	genesisBlockHash = "0f8fb4e17d2ab9f3097af75ca7fd16064160fb8043db94909e00dd4e257b9dc4"
+	// testContractHash is an LE hash of NEP-17 "Rubl" contract deployed at block #2
+	// of basic testing chain.
+	testContractHash = "449fe8fbd4523072f5e3a4dfa17a494c119d4c08"
+	// deploymentTxHash is an LE hash of transaction that deploys NEP-17 "Rubl"
+	// contract at block #2 of basic testing chain.
+	deploymentTxHash = "bbb8ec059dd320dc9de5a8fb8c75351d8e369fca0256f7a6bdb623dcf71861ed"
+	// verifyContractHash is an LE hash of "Verify" contract deployed at block #7 of
+	// basic testing chain.
+	verifyContractHash = "06ed5314c2e4cb103029a60b86d46afa2fb8f67c"
+	// verifyContractAVM is an LE hash of "Verify" contract deployed at block #7 of
+	// basic testing chain.
+	verifyContractAVM = "VwIAQS1RCDBwDBTunqIsJ+NL0BSPxBCOCPdOj1BIskrZMCQE2zBxaBPOStkoJATbKGlK2SgkBNsol0A="
+	// verifyWithArgsContractHash is an LE hash of "VerifyWithArgs" contract deployed
+	// at block #10 of basic testing chain.
+	verifyWithArgsContractHash = "6261b3bf753bdc3d24c1327a23fd891e1c8a7ccd"
+	// nnsContractHash is an LE hash of NEP-11 non-divisible "examples/nft-nd-nns"
+	// contract deployed at block #11 of basic testing chain.
+	nnsContractHash = "450d1918a72fef97b48096bfec8d749961deef55"
+	// nnsToken1ID is a hex-encoded ID of the first NEP-11 NNS token minted at block
+	// #14 of basic testing chain.
+	nnsToken1ID = "6e656f2e636f6d"
+	// nfsoContractHash is an LE hash of NEP-11 divisible "examples/nft-d" ("NeoFS
+	// Object") contract deployed at block #17 of basic testing chain.
+	nfsoContractHash = "914246ab7888ba4eb3ddebc9cb5433c2edcc1671"
+	// nfsoToken1ID is a hex-encoded ID of the first NEP-11 NFSO token minted at
+	// block #18 of basic testing chain.
+	nfsoToken1ID = "7e244ffd6aa85fb1579d2ed22e9b761ab62e3486"
+	// storageContractHash is an LE hash of "Storage" contract deployed at block #22
+	// of basic testing chain.
+	storageContractHash = "ebc0c16a76c808cd4dde6bcc063f09e45e331ec7"
+	// faultedTxHashLE is an LE hash of FAULTed transaction accepted at block #23 of
+	// basic testing chain.
+	faultedTxHashLE = "82279bfe9bada282ca0f8cb8e0bb124b921af36f00c69a518320322c6f4fef60"
+	// faultedTxBlock is the number of block of basic testing chain that contains
+	// FAULTed transaction.
+	faultedTxBlock uint32 = 23
+	// invokescriptContractAVM is a base64-encoded AVM of
+	// "pkg/internal/basicchain/testdata/invokescript_contract.go" contract that is
+	// not yet deployed to the testing basic chain.
+	invokescriptContractAVM = "VwIADBQBDAMOBQYMDQIODw0DDgcJAAAAAErZMCQE2zBwaEH4J+yMqiYEEUAMFA0PAwIJAAIBAwcDBAUCAQAOBgwJStkwJATbMHFpQfgn7IyqJgQSQBNA"
+	// block20StateRootLE is an LE stateroot of block #20 of basic testing chain.
+	block20StateRootLE = "394e20adf99a6ba160df7351770dfb193ee8af174b7b3ed45f4e2ae8c43694e8"
```

Comment:
Changed?

Reply:
Looks like a merge artifacts, I switched to `nep24` branch before. Fixed.